proto/kvrpcpb.proto

```diff
@@ -815,6 +815,12 @@ message ScanDetailV2 {
 
     // Total number of bytes from block reads.
     uint64 rocksdb_block_read_byte = 7;
+
+    // Total number of keys from the scan executor (i.e. TableScan or IndexScan) reads.
```

Comment:
BTW this value may be exactly the same as `processed_versions` (please confirm).

Reply:
ur right